.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+aaaaa
+aaaaaa
+aaaaaaaaaaaaaaaaaaa
+AAAAAABBBBBBCCC
+AAAAABBBBBBBCCC
+AAD
+ABANDONFONT
+abcd
+abcde
+abcdef
+ABCDEFG
+ABCDEFGH
+ABCDEFGHIJ
+abcdefghijk
+abcdefghijklmnop
+ABCDEFGHIJKLMNOPQRST
+abcdefghijklmnopqrstuvwxyz
+ABCG
+abe
+abec
+abf
+abi
+acb
+accd
+ACCEPTFILES
+ACCESSDENIED
+acec
+acf
+acidev
+ACIOSS
+ACover
+actctx
+ACTCTXW
+activatable
+ACTIVEBORDER
+ACTIVECAPTION
+ADDALIAS
+ADDREF
+addressof
+ADDSTRING
+ADDTOOL
+AEB
+aeed
+aef
+AEnd
+afae
+afceeeaa
+AFE
+AFEF
+AFew
+AFill
+AFX
+AHelper
+ahz
+AImpl
+AInplace
+akb
+ALIGNRIGHT
+alloc
+allocing
+alpc
+ALTERNATENAME
+ALTF
+ALTNUMPAD
+ALWAYSTIP
+amd
+ansd
+ansicode
+ansicpg
+ANSISYS
+ANSISYSRC
+ANSISYSSC
+antialias
+antialiasing
+ANull
+Anup
+anycpu
+AOn
+APARTMENTTHREADED
+APCs
+api
+APIENTRY
+apiset
+apos
+appconsult
+appdata
+APPICON
+appium
+applet
+appletname
+applicationmodel
+APPLMODAL
+APPLOGIC
+appmodel
+apps
+APPWINDOW
+appx
+appxbundle
+Appxmandifest
+appxmanifest
+appxsources
+APrep
+aps
+apsect
+APSTUDIO
+architected
+argb
+argc
+args
+argv
+arigatoo
+ARRAYSIZE
+ARROWKEYS
+ASBRST
+ASBSET
+ASDF
+asdfghjkl
+ASetting
+ASingle
+asm
+asmv
+asmx
+aspnet
+aspx
+astextplain
+AStomps
+ASYNCWINDOWPOS
+atch
+ATest
+atg
+ative
+ativecxapp
+atl
+atlmfc
+attr
+ATTRCOLOR
+aumid
+austdi
+Authenticode
+AUTOBUDDY
+AUTOCHECKBOX
+Autogenerated
+AUTOHSCROLL
+autologin
+automagically
+autopositioning
+AUTORADIOBUTTON
+autoscrolling
+Autowrap
+AVerify
+AVI
+awch
+azuredevopspodcast
+azurewebsites
+azzle
+baac
+backend
+backgrounded
+backgrounder
+backgrounding
+backport
+backstory
+Ballmer
+Batang
+baz
+Bazz
+BBAF
+BBBB
+BBBBBBBBBBBBBBBBBBBBBBBBBB
+BBBBCCCCC
+bbc
+bbd
+BBDM
+bbe
+bbggrr
+bbwe
+bc
+bca
+bcb
+bcc
+bcount
+bcrypt
+bcx
+bcz
+bda
+BDB
+BEB
+BEBE
+BEFOREPARENT
+beginthread
+bfb
+BFC
+BFEF
+Bfu
+bgcolor
+bgfx
+bgidx
+Bgk
+BGR
+BGRA
+BHID
+bhoj
+Bhojwani
```

Comment:
You know, it might be nice to have another whitelist file that is just "names of contributors and their aliases" and make the true whitelist be a union of the files.

Reply:
Hmm. That's probably doable. If whitelist is a directory instead of a file, I can cat them all together. It'll take a bit to reason through, but ... yeah...

Reply:
Yeah if it's a directory, I'd have like
`whitelist\everythingelse.txt`
`whitelist\names.txt`
`whitelist\t.txt`
`whitelist\n.txt`
`whitelist\r.txt`
`whitelist\README.md` <-- to explain WTF the files mean
or whatnot.